extensions/amp-sidebar/amp-sidebar.md

```diff
@@ -201,6 +201,46 @@ In the following example, we display a `toolbar` if the window width is less tha
 See live demos at [AMP By Example](https://ampbyexample.com/components/amp-sidebar/).
 {% endcall %}
 
+## Sidebar for Stories
+The sidebar is also usable within the `amp-story` [component](https://www.ampproject.org/stories/). However there are certain restrictions on what attributes and features are allowed. The following are **disallowed** attributes and features of sidebar within an `amp-story`:
+
+- side
+- open
+- toolbar
+- toolbar-target
+
+### Behavior
+- The `<amp-sidebar>` must be a direct child of `<amp-story>`
+- The sidebar defaults to the "start" side for regular AMP documents, meaning right for left-right languages and left for right-to-left languages
+- The `<amp-sidebar>` has default background color of white and is overridable in CSS
+- The `<amp-sidebar>` the maximum width of the sidebar is enforced (320px for the desktop experience and otherwise 280px)
```

Comment:
"The `<amp-sidebar>` the maximum width of the sidebar is enforced (320px for the desktop experience and otherwise 280px)"

to

"Maximum width of `<amp-sidebar>` is enforced at `280px` and at `320px` for desktop experiences.

Reply:
Done